.github/workflows/built-release.yml

```diff
@@ -0,0 +1,18 @@
+name: Built Release
+
+on:
+  push:
+    branches:
+      - develop
+      - main
+      - production
+
+jobs:
+  built-release:
+    uses: alleyinteractive/.github/.github/workflows/built-release.yml@main
+    if: ${{ github.repository != 'alleyinteractive/create-wordpress-plugin' }}
+    with:
+      node: 16
+      php: '8.2'
```

Comment:
This seems like it was copied over from create-wordpress-plugin. Just want to make sure these are the proper versions.

Reply:
Yes, those are correct, we have ticketed updating node for later.